app/__init__.py

```diff
@@ -74,6 +75,10 @@ def refresh_session():
     application.add_template_filter(question_references)
     application.add_template_filter(parse_document_upload_time)
 
+    @application.template_filter('convert_to_boolean')
+    def convert_to_boolean_filter(data):
+        return convert_to_boolean(data)
```

Comment:
Can this just be `application.add_template_filter(convert_to_boolean)` like the ones above?